<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTreeView" name="treeView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuCommands">
    <property name="title">
     <string>Commands</string>
    </property>
    <addaction name="actionSet_Nick"/>
    <addaction name="actionCreateChannel"/>
   </widget>
   <addaction name="menuView"/>
   <addaction name="menuCommands"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSet_Nick">
   <property name="text">
    <string>Set Nick...</string>
   </property>
  </action>
  <action name="actionCreateChannel">
   <property name="text">
    <string>Create Channel...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
